pandas/tests/tslibs/test_to_offset.py

```diff
@@ -194,7 +194,7 @@ def test_to_offset_lowercase_frequency_deprecated(freq_depr):
     depr_msg = f"'{freq_depr[1:]}' is deprecated and will be removed in a "
     f"future version, please use '{freq_depr.upper()[1:]}' instead."
 
-    with pytest.raises(FutureWarning, match=depr_msg):
+    with pytest.warns(FutureWarning, match=depr_msg):
```

Comment:
Please use `tm.assert_produces_warning` (and below)

Reply:
Done.